src/grid/interfaces.tsx

```diff
@@ -57,11 +57,13 @@ export namespace GridProps {
 
     /**
      * The number of grid elements to pull the column to the left by.
+     * @deprecated The use of this property is not recommended for accessibility reasons.
```

Comment:
I'd also add what they should use instead. 

"@deprecated The use of this property is not recommended for accessibility reasons. Use ... instead."